microceph/cmd/microceph/enable_mds.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"context"
+
+	"github.com/canonical/microceph/microceph/api/types"
+	"github.com/canonical/microceph/microceph/client"
+	"github.com/canonical/microcluster/microcluster"
+	"github.com/spf13/cobra"
+)
+
+type cmdEnableMDS struct {
+	common     *CmdControl
+	wait       bool
+	flagTarget string
+}
+
+func (c *cmdEnableMDS) Command() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "mds [--target <server>]",
+		Short: "Enable the MDS service on the --target server (default: this server)",
+		RunE:  c.Run,
+	}
+	cmd.PersistentFlags().StringVar(&c.flagTarget, "target", "", "Server hostname (default: this server)")
+	cmd.Flags().BoolVar(&c.wait, "wait", false, "Wait for mds service to be up.")
```

Comment:
Wondering if we shouldn't make `--wait=true` the default here. 

If I understand correctly, with `--wait=false` users won't see any errors unless they look in the log -- which isn't something a user would do routinely.

Otoh, any errors that might occur are directly reported back when `--wait=true` which feels like the safer and more explicit option. 

Same for the commands below

Reply:
Hmm, that does convince me to use wait=true for such instances.